sources/packages/backend/apps/queue-consumers/src/processors/schedulers/esdc-integration/ecert-integration/_tests_/parsers/part-time-e-cert-record-parser.ts

```diff
@@ -28,4 +28,12 @@ export class PartTimeCertRecordParser extends ECertRecordParser {
   get lastName(): string {
     return this.record.substring(2, 27).trim();
   }
+
+  /**
+   * Disbursement amount.
+   * This field includes 2 decimals in the file.
+   */
+  get disbursementAmount(): number {
+    return +this.record.substring(457, 466).trim() / 100;
```

Comment:
The idea is to have these values as raw as possible to allow the validation of the raw values themselves.
For this PR, validating the extra 2 decimals in the E2E is the desirable test.
Even the `trim` is debatable but since it is already in the other ones we can leave for now.